jetson/percep/meson.build

```diff
@@ -61,6 +78,7 @@ conf_data.set10('OBSTACLE_RECORD', obs_record)
 conf_data.set10('ZED_SDK_PRESENT', with_zed)
 conf_data.set10('PERCEPTION_DEBUG', perception_debug)
 conf_data.set10('WRITE_CURR_FRAME_TO_DISK', write_frame)
+conf_data.set10('VM', vm)
```

Comment:
NIT: Look at the style of the other key value pairs here. They are very explicit as to what they represent, you should do the same with VM.

Reply:
Fixed